CONTRIBUTING.md

```diff
@@ -18,14 +18,14 @@ This repository contains the conceptual documentation for .NET. The .NET documen
 - [API reference](https://github.com/dotnet/dotnet-api-docs)
 - [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
 
-Issues and tasks for all these repositories are tracked here.
```

Comment:
This text is intended to apply to all of the previously listed repositories, except that it's not accurate. Better would be to list individually where each of the issues and tasks are tracked. For example:
- [API reference](https://github.com/dotnet/dotnet-api-docs). Issues and tasks for this repo are tracked [in the [dotnet/dotnet-api-docs repo](https://github.com/dotnet/dotnet-api-docs).
- [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs). Issues and tasks for this repo are tracked in the [dotnet/docs repo](https://github.com/dotnet/docs/issues).
-  [Code samples and snippets](https://github.com/dotnet/samples). Issues and tasks for this repo are tracked in the [dotnet/docs repo](https://github.com/dotnet/docs/issues).

Reply:
What about <https://github.com/dotnet/samples/issues> ?

Reply:
We had to open that for the dependabot to work, but we would still prefer if folks open the issue here since it's part of our daily workflow.

Reply:
Wondering if we should also add repos such as roslyn-api-docs and ml-api-docs to this list. Issues for roslyn right now are also filed in dotnet/docs. Not sure where people go to file ML.NET API docs issues. @natke?